content/articles/2018-07-ggplot2-3-0-0.Rmarkdown

```diff
@@ -0,0 +1,173 @@
+---
+title: ggplot2 3.0.0
+author: Mara Averick
+date: '2018-07-05'
+slug: ggplot2-3-0-0
+description: > 
+  ggplot2 3.0.0 is on CRAN.
+categories:
+  - package
+photo:
+  url: https://unsplash.com/photos/wEL2zPX3jDg
+  author: Fabio Ballasina
+---
+
+```{r setup, include = FALSE}
+library(ggplot2)
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>", dpi = 144, fig.width = 5, fig.height = 4)
+```
+
+We're well chuffed to announce the release of ggplot2 3.0.0. This is a major release: it was previously announced as ggplot2 2.3.0, but we decided to increment the version number because there are some big changes under the hood. Most importantly ggplot2 now supports tidy evaluation, which makes it easier to programmatically build plots with ggplot2 in the same way you can programmatically build data manipulation pipelines with dplyr.
+
+Install ggplot2 with:
+
+```{r install, eval = FALSE}
+install.packages("ggplot2")
+```
+
+## Breaking changes
+
+As described in our [pre-release post](https://www.tidyverse.org/articles/2018/05/ggplot2-2-3-0/), we decided 
```

Comment:
This feels a little too assertive. Can we make a bit more empathetic? 

Maybe something like: In stable, long-standing packages like ggplot2, we put in a lot of effort to make sure that we don't introduce backward incompatible changes. For this release, that involved testing on five versions of R (3.1 - 3.5) writing a new visual testing package ([link to vdiffr]), running R CMD check on downstream packages (over 6000 times! [link to https://www.tidyverse.org/articles/2018/05/ggplot2-2-3-0/#our-process]), and widely advertising the release to get the community's help. However, sometimes we do decide that it's worth breaking ...

(and then remove the next paragraph)